vec/vec-common/src/main/java/com/foursoft/harness/vec/common/util/DateUtils.java

```diff
@@ -53,40 +56,18 @@ private DateUtils() {
      * @throws VecException In case the conversion fails.
      */
     public static XMLGregorianCalendar currentDate() {
-        final Instant date = Instant.now();
-        return toXMLGregorianCalendar(date.toString());
+        return toXMLGregorianCalendar(Instant.now());
```

Comment:
@DManstrator could you please add the clock overwrite code I wrote last week and use the clock here